.github/workflows/snapshot-release.yml

```diff
@@ -76,8 +76,13 @@ jobs:
       - name: Bump Package Versions
         id: changesets
         run: |
-          pnpm exec changeset version --snapshot ${{ steps.getSnapshotName.outputs.result }} > changesets.output.txt 2>&1
-          echo ::set-output name=result::`cat changesets.output.txt`
+          pnpm exec changeset status --output changesets.output.json 2>&1
+          pnpm exec changeset version --snapshot ${{ steps.getSnapshotName.outputs.result }} > changesets-version.output.txt 2>&1
+          {
+            echo 'result<<__EOF'
+            cat changesets.output.json
+            echo __EOF
+          } >> "$GITHUB_OUTPUT"
```

Comment:
I don't know this syntax :O (the curly brackets). What does it do?

Reply:
I copied this from the GitHub docs but it doesn't seem to work. It was supposed to run `>> $GITHUB_OUTPUT` after each action, but it seems invalid. Switched this to a more traditional syntax.